gcloud/storage/bucket.py

```diff
@@ -50,7 +50,7 @@ def get_items_from_response(self, response):
         """
         self.prefixes = tuple(response.get('prefixes', ()))
         for item in response.get('items', []):
-            yield Blob(properties=item, bucket=self.bucket)
+            yield Blob(None, properties=item, bucket=self.bucket)
```

Comment:
From @tseaver on previous commit:

I guess we are guaranteed that properties will have a name?